.huskyrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "hooks": {
+    "post-checkout": "yarn install --frozen-lockfile",
```

Comment:
Maybe use HUSKY_GIT_PARAMS to check if git is checking out a branch (see https://youtu.be/ikn_dBSski8?t=780 for details)?

Reply:
Better keep consistent the way we use those dep between our project. I include it in the package.json then :+1:

Reply:
Good idea!